dessia_common/workflow/core.py

```diff
@@ -1941,6 +1943,138 @@ def _method_jsonschemas(self):
         warnings.warn("method_jsonschema method is deprecated. Use method_schema instead", DeprecationWarning)
         return self.method_schemas
 
+    @staticmethod
+    def instantiate_objects(objects):
```

Comment:
Why does need to be a workflow method ?
In addition to that, method name seems off

Reply:
done !